doc/Extensions/Applications.md

```diff
@@ -1002,6 +1002,123 @@ You may need to configure `$server` or `$port`.
 
 Verify it is working by running `/usr/lib/check_mk_agent/local/gpsd`
 
+## HTTP Access Log Combined
+
+### SNMP Extend
+
+1. Download the script onto the desired host.
+```
+wget https://raw.githubusercontent.com/librenms/librenms-agent/master/snmp/http_access_log_combined -O /etc/snmp/http_access_log_combined
+```
+
+2. Make the script executable
+```
+chmod +x /etc/snmp/http_access_log_combined
+```
+
+3. Install the depends
+```
+# FreeBSD
+pkg install p5-File-Slurp p5-MIME-Base64 p5-JSON p5-Statistics-Lite p5-File-ReadBackwards
+
+# Debian
+apt-get install libfile-slurp-perl libmime-base64-perl libjson-perl libstatistics-lite-perl libfile-readbackwards-perl
+```
+
+4. Configure it if neeeded. Uses
+   `/usr/local/etc/http_access_log_combined_extend.json`, unless
+   specified via `-c`. See further below for configuration
+   information.
+
+5. If on large setups where it won't complete in a timely manner, run
+   it via cron.
+```
+4/5 * * * * root /etc/snmp/http_access_log_combined -b -q -w
```

Comment:
Is this supposed to be 4/5?

Reply:
Hmm, yeah, this should be */5 for more generic cron. Being able to do 4/5 is a FreeBSD thing.